## 
1.
2.try()catch(){} 异常打印不了。